### PR TITLE
mockup-mogo - task 8 JS

### PR DESCRIPTION
**Pull request by maycatsby**
https://maycatsby.github.io/ui-internship-lwo2/HTML/mockup-mogo/

### CODE REVIEW

File: HTML/mockup-mogo/js/task8.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import {promiseElements} from './elements.js';
+
+export const showPopup = () => {
```
Comment (tsvetkovv) [inline HTML/mockup-mogo/js/task8.js:3]: Call `showPopup` means pop up will be shown. But this function just initialise popup, so `initPopup` is more relevant

File: HTML/mockup-mogo/js/task8.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  promiseElements.then((data) => {
+    const {popup, portfolioContainer, popupImage} = data;
+    popup.addEventListener('click', () => {
+      popup.classList.remove('u-show');
+    });
+    portfolioContainer.addEventListener('click', (e) => {
+      let clickTarget = e.target.closest('.picture').dataset.image;
```
Comment (tsvetkovv) [inline HTML/mockup-mogo/js/task8.js:10]: `const`?

**Review by tsvetkovv — APPROVED** [review]
good

**Review by tkachukivan — APPROVED** [review]
not very useful for now due to browser support but just take a look at [dialog tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog)

**Pull request merged**